core/src/test/scala/unit/kafka/server/BaseRequestTest.scala

```diff
@@ -44,6 +44,7 @@ abstract class BaseRequestTest extends IntegrationTestHarness {
   protected def brokerPropertyOverrides(properties: Properties): Unit = {}
 
   override def modifyConfigs(props: Seq[Properties]): Unit = {
+    super.modifyConfigs(props)
```

Comment:
If we can make the ParallelControllerStartupTest inherit from IntegrationTestHarness, this change won't be needed in this PR.

Reply:
That's true, but it's still a valid bugfix.  (Then again, I have a followup test fix to do for PlaintextAdminIntegrationTest, so I could include it with that instead.)

Reply:
Deferred to followup PR.